### PR TITLE
Minor formatting changes to Expectations section

### PR DESCRIPTION
**Pull request by SameeraKamra**
Added ;(and | or) at the end of the list:
For each test target, the [outcome](#outcome) of at least one of the following rules is passed:

- [`Video` Element Visual Content Has Audio Description](https://act-rules.github.io/rules/1ea59c);(and | or)
- [`Video` Element Visual Content Has Transcript](https://act-rules.github.io/rules/1a02b0);(and | or)
- [`Video` Element Visual Content Has Description Track](https://act-rules.github.io/rules/f196ce);(and | or)
- [`Video` Element Content Is Media Alternative For Text](https://act-rules.github.io/rules/ab4d13)

Need for Final Call:
No need for Final Call
---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 1]
```diff
@@ -66,9 +66,9 @@ This rule applies to every [non-streaming](#non-streaming-media-element) `video`
 
 For each test target, the [outcome](#outcome) of at least one of the following rules is passed:
 
-- [`Video` Element Visual Content Has Audio Description](https://act-rules.github.io/rules/1ea59c)
-- [`Video` Element Visual Content Has Transcript](https://act-rules.github.io/rules/1a02b0)
-- [`Video` Element Visual Content Has Description Track](https://act-rules.github.io/rules/f196ce)
+- [`Video` Element Visual Content Has Audio Description](https://act-rules.github.io/rules/1ea59c);(and | or)
```
Comment (ajanec01) [inline _rules/video-alternative-for-visual-c5a4ea.md:69]:
```suggestion
- [`Video` Element Visual Content Has Audio Description](https://act-rules.github.io/rules/1ea59c); or
```

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 2]
```diff
@@ -66,9 +66,9 @@ This rule applies to every [non-streaming](#non-streaming-media-element) `video`
...
+- [`Video` Element Visual Content Has Transcript](https://act-rules.github.io/rules/1a02b0);(and | or)
```
Comment (ajanec01) [inline _rules/video-alternative-for-visual-c5a4ea.md:70]:
```suggestion
- [`Video` Element Visual Content Has Transcript](https://act-rules.github.io/rules/1a02b0); or
```

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 3]
```diff
@@ -66,9 +66,9 @@ This rule applies to every [non-streaming](#non-streaming-media-element) `video`
...
+- [`Video` Element Visual Content Has Description Track](https://act-rules.github.io/rules/f196ce);(and | or)
```
Comment (ajanec01) [inline _rules/video-alternative-for-visual-c5a4ea.md:71]:
```suggestion
- [`Video` Element Visual Content Has Description Track](https://act-rules.github.io/rules/f196ce); or
```

**Review by ajanec01 — CHANGES_REQUESTED** [review]
Hey Sameera.
Something I didn't mention before, it's a good practice to refer to the rule you're editing in the pull requests title or the description.

**Review by ajanec01 — APPROVED** [review]

**Comment by SameeraKamra** [comment]
Please suggest the next steps on this pull request. Thanks!

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 4]
```diff
@@ -66,9 +66,9 @@ This rule applies to every [non-streaming](#non-streaming-media-element) `video`
 
 For each test target, the [outcome](#outcome) of at least one of the following rules is passed:
 
-- [`Video` Element Visual Content Has Audio Description](https://act-rules.github.io/rules/1ea59c)
-- [`Video` Element Visual Content Has Transcript](https://act-rules.github.io/rules/1a02b0)
-- [`Video` Element Visual Content Has Description Track](https://act-rules.github.io/rules/f196ce)
+- [`Video` Element Visual Content Has Audio Description](https://act-rules.github.io/rules/1ea59c); or
+- [`Video` Element Visual Content Has Transcript](https://act-rules.github.io/rules/1a02b0); or
+- [`Video` Element Visual Content Has Description Track](https://act-rules.github.io/rules/f196ce); or
 - [`Video` Element Content Is Media Alternative For Text](https://act-rules.github.io/rules/ab4d13)
```
Comment (Jym77) [inline _rules/video-alternative-for-visual-c5a4ea.md:72]:
```suggestion
- [`Video` Element Content Is Media Alternative For Text](https://act-rules.github.io/rules/ab4d13).
```
Last one should end up with full period.

**Review by Jym77 — CHANGES_REQUESTED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Review by colabottles — APPROVED** [review]

**Review by kherr-a11y — COMMENTED** [review]
Agreed

**Pull request merged**